staging/src/k8s.io/client-go/util/jsonpath/node.go

```diff
@@ -139,12 +139,14 @@ type ParamsEntry struct {
 type ArrayNode struct {
 	NodeType
 	Params [3]ParamsEntry // start, end, step
+	Value  string
```

Comment:
I do not see this additional field being read anywhere in the diff (or any of the other similarly new `Value` fields)